app/assets/javascripts/selectors/backlog.js

```diff
@@ -43,10 +43,16 @@ const sortStartedStories = (stories) => {
     .sort(Story.compareStartedAt);
 }
 
-export const groupStoriesInSprints = (stories, project, initialSprintNumber, pastIterations) =>
+export const groupStoriesInSprints = (stories, project, initialSprintNumber) =>
   Iteration.groupBySprints(
     stories,
     project,
     initialSprintNumber,
-    pastIterations
   );
+
+export const getSprintVelocity = (project, pastIterations) => 
```

Comment:
I believe you can refactor it for:
```js
export const getSprintVelocity = Iteration.sprintVelocity;
```

Reply:
sorry, I didn't understand, could you explain ? :confused:

Reply:
I've checked all methods from this file and `groupStoriesInSprints` and `getSprintVelocity` are calling a function with the same parameters. In other words, `groupStoriesInSprints` is like calling `Iteration.groupBySprints`, and `getSprintVelocity` is like calling `Iteration.sprintVelocity`.
```js
export const groupStoriesInSprints = Iteration.groupBySprints;
export const getSprintVelocity = Iteration.sprintVelocity;
```

So, my question now is: why don't you call the method `Iteration.groupBySprints` instead of creating a new method `groupStoriesInSprints`? (same question for `getSprintVelocity`) 🤔

Reply:
Since it is not your code and you are just refactoring, there is no problem just refactor these two methods to:
```js
export const groupStoriesInSprints = Iteration.groupBySprints;
export const getSprintVelocity = Iteration.sprintVelocity;
```
😄

Reply:
okay :D